src/main/factories/create.ts

```diff
@@ -1,6 +1,6 @@
 import { BrowserWindow, BrowserWindowConstructorOptions, session } from 'electron';
 
-import { APP_CONFIG } from '@/app.config';
+import { APP_CONFIG } from '../../../app.config';
```

Comment:
why change it from `@/app.config` ? Let's have it as it was

Reply:
This alias was not working for electron
Requires research